var/spack/repos/builtin/packages/qucs/package.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+import os
+
+
+class Qucs(AutotoolsPackage):
+    """QUCS - Quite Universal Circuit Simulator:
+       Qucs is an integrated circuit simulator which means you are able to
+       setup a circuit with a graphical user interface (GUI) and simulate
+       the large-signal, small-signal and noise behaviour of the circuit.
+       After that simulation has finished you can view the simulation results
+       on a presentation page or window.
+       This is the complete environment as provided at Sourceforge, the main repository.
+    """
+
+    homepage = "http://qucs.sourceforge.net/"
+    url      = "https://sourceforge.net/projects/qucs/files/qucs/0.0.19/qucs-0.0.19.tar.gz"
+    git      = "https://git.code.sf.net/p/qucs/git"
+
+    version('master', branch='master')
+    version('0.0.19', sha256='45c6434fde24c533e63550675ac21cdbd3cc6cbba29b82a1dc3f36e7dd4b3b3e')
+    version('0.0.18', sha256='3609a18b57485dc9f19886ac6694667f3251702175bd1cbbbea37981b2c482a7')
+
+    # Can use external simulators:
+    variant(
+        'simulators', default='qucs', multi=True,
+        values=('qucs', 'ngspice', 'xyce'),
+        description='Simulators'
+    )
```

Comment:
Just to double check, Qucs can support multiple of these backend at once, correct?

Reply:
Yes sure ; so that then the user can choose which simulator he wants, so he can compare as well.